v2-migration/src/main/java/software/amazon/awssdk/v2migration/NewClassToBuilder.java

```diff
@@ -104,7 +105,8 @@ public J visitNewClass(J.NewClass previousNewClass, ExecutionContext executionCo
 
             JavaType.FullyQualified classType = (JavaType.FullyQualified) newClass.getType();
 
-            if (!isEligibleToConvertToBuilder(classType)) {
+            if (!isEligibleToConvertToBuilder(classType) ||
+                !newClass.getConstructorType().getParameterNames().isEmpty()) {
```

Comment:
Added this to avoid converting ctor that has parameters to empty builder, which is incorrect